fhir-install/src/main/docker/ibm-fhir-term-graph-loader/Dockerfile

```diff
@@ -4,25 +4,62 @@
 # SPDX-License-Identifier: Apache-2.0
 # ----------------------------------------------------------------------------
 
-# Build stage
-FROM maven:3.6.0-jdk-11-slim AS build
-COPY . /
+# Stage: Base
 
-RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
-RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
-RUN mvn -f /fhir-examples/pom.xml install -DskipTests=true
+# IBM Semeru Runtimes provides Non-official docker images as part of this repo. These are maintained by IBM.
```

Comment:
provide unofficial docker images...

Reply:
it's a copy of the semeru runtime statement.